packages/driver/test/cypress/integration/commands/actions/type_spec.js

```diff
@@ -4340,7 +4340,11 @@ describe('src/cy/commands/actions/type', () => {
 
           const allChars = _.keys(Keyboard.specialChars).concat(_.keys(Keyboard.modifierChars)).join(', ')
 
-          expect(err.message).to.eq(`Special character sequence: '{bar}' is not recognized. Available sequences are: ${allChars}`)
+          expect(err.message).to.eq(`Special character sequence: '{bar}' is not recognized. Available sequences are: ${allChars}
```

Comment:
Approving this PR but this would be a good place to introduce template tags using the `common-tags` module.